include/picongpu/particles/collision/detail/ListEntry.hpp

```diff
@@ -0,0 +1,292 @@
+/* Copyright 2014-2021 Rene Widera, Pawel Ordyna
```

Comment:
Is 2014 right? (it may be, i just don't know so wondering)

Reply:
@psychocoderHPC should it be 2019?

Reply:
I think I copy-pasted the header from somewhere, yes should be `2019 - 2021`